content/references/rippled-api/admin-rippled-methods/logging-and-data-management-methods/node_to_shard.md

```diff
@@ -0,0 +1,122 @@
+---
+html: node_to_shard.html
+parent: logging-and-data-management-methods.html
+blurb: Copy data from the ledger store into the shard store.
+labels:
+  - Data Retention
+---
+# node_to_shard
+[[Source]](https://github.com/undertome/rippled/blob/develop/src/ripple/rpc/handlers/NodeToShard.cpp "Source")
```

Comment:
This source address should probably be moved to the main rippled repository?  Possibly this: https://github.com/ripple/rippled/blob/develop/src/ripple/rpc/handlers/NodeToShard.cpp